app/views/shared/_directory.html.erb

```diff
@@ -8,4 +8,11 @@
     <p class="directory__description"><%= item.description %></p>
   </div>
   <% end %>
+  <div class="directory__item">
+    <%= heading_tag(level: heading_level ||= 3, class: 'directory__title') do %>
```

Comment:
I've copied the exact same markup used to produce the other links so they're all styled the same.